test/order_test.rb

```diff
@@ -142,20 +142,36 @@
 
     it "Returns accurate information about the last order" do
       # TODO: Your test code here!
+      last = Order.all.last
+      puts last.inspect
```

Comment:
`puts` is great to use during your development process. Minor note: you should remove this when the project is complete.